packages/components/src/Components/BulkSelect/BulkSelect.js

```diff
@@ -49,6 +49,7 @@ visible unless you update it.');
                                                 aria-label="Select all"
                                                 onChange={ onSelect }
                                                 checked={ checked }
```

Comment:
Do we need both `checked` and `isChecked` ?

Reply:
Yeah, I wanted to keep the checked for consistency reasons, but it works same way without it.